src/Integrations/Moments/SettingsMoments.php

```diff
@@ -98,6 +126,143 @@ public function isSettingsGlobalValid(): bool
+			[
+				'component' => 'tabs',
+				'tabsContent' => [
+					[
+						'component' => 'tab',
+						'tabLabel' => \__('General', 'eightshift-forms'),
+						'tabContent' => [
+							[
+								'component' => 'checkboxes',
+								'checkboxesFieldLabel' => '',
+								'checkboxesName' => UtilsSettingsHelper::getSettingName(self::SETTINGS_MOMENTS_USE_EVENTS_KEY),
+								'checkboxesContent' => [
+									[
+										'component' => 'checkbox',
+										'checkboxLabel' => \__('Use Events', 'eightshift-forms'),
+										'checkboxHelp' => \__('Send events to Moments', 'eightshift-forms'),
+										'checkboxIsChecked' => $useEvents,
+										'checkboxValue' => self::SETTINGS_MOMENTS_USE_EVENTS_KEY,
+										'checkboxSingleSubmit' => true,
+										'checkboxAsToggle' => true,
+									]
+								]
+							],
+							...($useEvents ? [
+								[
+									'component' => 'divider',
+									'dividerExtraVSpacing' => true,
+								],
+								[
+									'component' => 'select',
+									'selectName' => UtilsSettingsHelper::getSettingName(self::SETTINGS_MOMENTS_EVENTS_EMAIL_FIELD_KEY),
+									'selectFieldHelp' => \__('You must select what field is used as an e-mail.', 'eightshift-forms'),
+									'selectFieldLabel' => \__('E-mail field', 'eightshift-forms'),
+									'selectPlaceholder' => \__('Select email field', 'eightshift-forms'),
+									'selectIsRequired' => true,
+									'selectContent' => \array_map(
+										static function ($option) use ($eventsEmailFieldValue) {
+											return [
+												'component' => 'select-option',
+												'selectOptionLabel' => $option,
+												'selectOptionValue' => $option,
+												'selectOptionIsSelected' => $eventsEmailFieldValue === $option,
+											];
+										},
+										$formFields
+									),
+								],
+								[
+									'component' => 'input',
+									'inputName' => UtilsSettingsHelper::getSettingName(self::SETTINGS_MOMENTS_EVENTS_EVENT_NAME_KEY),
+									'inputFieldLabel' => \__('Event name', 'eightshift-forms'),
+									// translators: %s will be replaced with forms field name and filter output copy.
+									'inputFieldHelp' => \__('Set event name used in the Moments integrations.', 'eightshift-forms'),
+									'inputType' => 'text',
+									'inputIsRequired' => true,
+									'inputValue' => $eventsEventNameValue,
+								],
+								...(($eventsEmailFieldValue && $eventsEventNameValue) ? [
+									[
+										'component' => 'divider',
+										'dividerExtraVSpacing' => true,
+									],
+									[
+										'component' => 'group',
+										'groupName' => UtilsSettingsHelper::getSettingName(self::SETTINGS_MOMENTS_EVENTS_MAP_KEY),
+										'groupSaveOneField' => true,
+										'groupStyle' => 'default-listing',
+										'groupContent' => [
+											[
+												'component' => 'field',
+												'fieldLabel' => '<b>' . \__('Moments event key', 'eightshift-forms') . '</b>',
```

Comment:
I suppose you are not using `esc_html__`  here so that translators cannot omit the <b>